stdlib/public/core/Bool.swift

```diff
@@ -100,6 +100,12 @@ public struct Bool {
   ///         print("Maybe another try?")
   ///     }
   ///
+  /// - Note: The algorithm used to create random values may change in a future
+  ///   version of Swift. If you're using a seeded generator and you need
```

Comment:
It's obvious to us, but perhaps we should be a bit more precise with the wording here. If the algorithm changes, values will not be reproducible whether you use a seeded generator or not. Something like:
~~~~
The algorithm used to create random values from a generator may change in a future version of Swift. If you need a program to always create the same sequence of random values, even when compiled using different versions of Swift, you need to make this mapping from the generator to random values part of your program as well, rather than relying on the standard library implementation.
~~~~

Reply:
I think these notes are the wrong place to put that information. They're meant to be a brief word of caution, not a place to expand on how you might work around it.

The extra level of detail in your proposed note here doesn't go far enough to be actionable. That's understandable given that it's in a note, but IMO that's an argument that the information is out of place.

Reply:
@krilnon The extra level of detail I'm proposing is mainly about being explicit that this does not only effect *seedable* generators. It's true no matter what generator someone uses.

Reply:
I'm not sure I'm following — how would I ever get reproducible values from a non-seeded generator, independent of the algorithm change? If I can't provide a seed, I'm not going to be able to have any expectation about what series of values I get.

Reply:
(a) users may not know what "seedable" means here, since we don't provide any in the standard lib.
(b) people will implement reproducible generators that do not seed (they just always seed with some default when initialized).

Given that, I don't see any reason to restrict the guidance to "seedable" generators. It's just an extra layer of complexity.

Reply:
@stephentyrone Thanks again for your comments here — I used some of your language, and focused on what they'll expect vs what might happen, rather than providing the solution. In particular, there's not an easily describable solution for things like `shuffle()`.